ui/app/models/pki/tidy.js

```diff
@@ -105,54 +121,27 @@ export default class PkiTidyModel extends Model {
   })
   tidyRevokedCerts;
 
-  /*
-  NOT IN DOCS - check with crypto
-  @attr('string') acme_account_safety_buffer;
-  @attr('boolean', { defaultValue: false }) maintain_stored_certificate_counts;
-  @attr('boolean', { defaultValue: false }) publish_stored_certificate_count_metrics;
-  @attr('boolean', { defaultValue: false }) tidy_acme;
-  */
-
   get useOpenAPI() {
     return true;
   }
   getHelpUrl(backend) {
     return `/v1/${backend}/config/auto-tidy?help=1`;
   }
 
-  get displayGroups() {
-    const groups = [
```

Comment:
throughout PKI there's a ton of places we have to update pki fields and things, so reusing these from the `this.sharedFields` attr